include/access_management.js

```diff
@@ -147,7 +147,8 @@ module.exports = function(pb) {
                 'ACCESS_ADMINISTRATOR': ls.get('ACCESS_ADMINISTRATOR'),
             };
         }
-        return null;
+        // Return an empty object instead of null, so things won't break after return
+        return {};
```

Comment:
What code exactly?  That code that calls should be providing the Localization instance.  Returning null was intentional to highlight code that used the call incorrectly.